rwandaopensource/backend.opensource.org.rw
# implement github login redirect

#### What does this PR do?
`implement GitHub login redirect`
`writing some tests`
#### Description of Task to be completed?
after the user login successfully with GitHub they should be redirected back to the frontend with the jwt-token
#### How should this be manually tested?
browse to `http://localhost:5000/auth/github`  while your frontend server is open or check the tests under `github.aouth.test.ts` 
#### Any background context you want to provide?
`don't forget to add new variables defined in .env.sample`
